### PR TITLE
Multi language

### PR DESCRIPTION
**Pull request by hellovietnam93**

### CODE REVIEW

**Comment by trantube94** [comment]
OK

**Comment by minhld1709** [comment]
OK

File: config/application.rb [hunk 1]
```diff
@@ -30,6 +30,18 @@ class Application < Rails::Application
...
+    # Do not swallow errors in after_commit/after_rollback callbacks.
     config.active_record.raise_in_transactional_callbacks = true
+    config.action_view.embed_authenticity_token_in_remote_forms = true
+
+    config.i18n.load_path += Dir[Rails.root.join("config", "locales", "**",
+      "*.{rb,yml}")]
+
+    config.i18n.available_locales = [:en, :jp]
+    config.action_view.embed_authenticity_token_in_remote_forms = true
+    config.filter_parameters += [:sync_token]
```
Comment (dieunb) [inline config/application.rb]: what for?

File: config/routes.rb [hunk 2]
```diff
@@ -1,56 +1,5 @@
 Rails.application.routes.draw do
-  # The priority is based upon order of creation: first created -> highest priority.
-  # See how all your routes lay out with "rake routes".
+  scope "(:locale)", locale: /#{I18n.available_locales.join("|")}/ do
```
Comment (dieunb) [inline config/routes.rb]: thinking about different ways!

**Pull request merged**